config/connectors.yml

```diff
@@ -1,7 +1,7 @@
 # general metadata
-version: "8.2.0-1648721082"
-repository: "git@github.com:elastic/connectors.git"
-revision: "48e7312286983c0197f7c3027487c4bfd6833090"
+version: "8.3.0.0"
```

Comment:
it feels weird to me to have these diffs included. Won't this start leading to merge conflicts all the time?

Reply:
It was recently removed from `.gitignore` to allow building the gem dependency from git to work in Enterprise Search. I don't think I like this approach, but that's where it came from.

Reply:
Yeah, before https://github.com/elastic/connectors/pull/62 was merged, this file was in `.gitignore`. I'd prefer we put it back there to avoid these diffs and to avoid accidentally committing an API key.